### ISSUE TITLE
Token starts returning AccessDenied eventually

### ISSUE THREAD

**Issue by bojanrajkovic** [pdlove/homebridge-smartthings#89]
After about 24-36 hours, SmartThings starts returning AccessDenied, despite the JSON API Complete SmartApp not being reconfigured at all. 

The only way to fix it is to re-do the setup and copy the new configuration. Obviously, this is less than ideal. Is there some way to debug why the token is going away?

The error message itself looks like this, and happens every 60 seconds as SmartThings refreshes:

```
00:18:08 - [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
00:19:08 - [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
00:20:08 - [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
00:21:08 - [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
00:22:08 - [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
00:23:08 - [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
```

**Comment by bojanrajkovic** [comment]
Oh, I've restarted HomeBridge with the `-D` flag after updating the configuration, to see if that will elucidate anything.

**Comment by pdlove** [comment]
I'm had a different issue last weekend that was server related. If you get it again, let me know and we'll step through how you're doing the setup.

**Comment by bojanrajkovic** [comment]
Makes sense. I'll keep an eye on it once I get home today. My hub was acting up last night, so that might have had something to do with it as well.

**Comment by rhearob** [comment]
I am getting similar errors during a clean install:

[9/17/2016, 11:18:49 PM] [SmartThings] Error received type AccessDenied - This request is not authorized by the specified access token
Sep 17 23:18:49 ae35.rheahometn.com homebridge[3957]: [9/17/2016, 11:18:49 PM] [SmartThings] Unknown Capabilities: []

I also noticed that when I set up the API in the smart things app, that it seems to not save any devices.  Every time I go back into the app it shows no devices selected.  Could this be related to the recent security update?
